cmake/configs/default.cmake

```diff
@@ -84,6 +84,7 @@ hunter_default_version(Imath VERSION 3.1.1)
 hunter_default_version(Immer VERSION 0.6.2-cf44615)
 hunter_default_version(Jpeg VERSION 9c-p0)
 hunter_default_version(JsonSpirit VERSION 0.0.4-hunter)
+hunter_default_version(jwt-cpp VERSION 0.5.2)
```

Comment:
Looks like the alphabetical order checker isn't happy. Can you move this to the lower-case list later in this file?